micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/Login/login.js

```
@@ -140,23 +141,30 @@ const Login = ({ config: propsConfig, t, isDisabled, loginOTPBased }) => {
   const onForgotPassword = () => {
     history.push(`/${window?.contextPath}/employee/user/forgot-password`);
   };
+  const defaultTenant = Digit.ULBService.getStateId();
   const defaultValue = {
-    code: Digit.ULBService.getStateId(),
-    name: Digit.Utils.locale.getTransformedLocale(`TENANT_TENANTS_${Digit.ULBService.getStateId()}`),
+    code: defaultTenant,
+    name: Digit.Utils.locale.getTransformedLocale(`TENANT_TENANTS_${defaultTenant}`),
   };
 
   let config = [{ body: propsConfig?.inputs }];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'config' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)